src/robot/utils/robottypes.py

```diff
@@ -12,9 +12,26 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+import sys
+
+PY3 = sys.version_info[0] == 3
+
+from .platform import IRONPYTHON
 from collections import Mapping
-from UserDict import UserDict
-from UserString import UserString
+if PY3:
+    from collections import UserList, UserDict, UserString
+else:
+    from UserList import UserList
+    from UserDict import UserDict
+    from UserString import UserString, MutableString
+if PY3:
+    long = int
+    bytes = bytes
+    unicode = str
+else:
+    long = long
+    bytes = str if not IRONPYTHON else bytes
+    unicode = unicode
```

Comment:
Why are these created and exposed? I don't see them used anywhere in the code base.